### DISCUSSION TITLE
Problem uninstalling the rocketchat windows exe application

### DISCUSSION THREAD

**Discussion by Seyilu** [RocketChat/Rocket.Chat#33147] [General]
Hello, I would like to report that uninstallation does not work in the next version of rocket 4.0.x. Only tested from version 4.0.0 and up. Namely: 
When you delete an application in the control panel, the application still remains in the folder and can be restored after deletion. We can log in normally etc. PLN 5 for this. Regards.

**Comment by reetp** [comment]
Please follow this guide on how and where to ask for support:

https://gist.github.com/reetp/a66149d5f060f260643a353ca7067a98#file-readme-md

Thanks.
